.circleci/config.yml

```diff
@@ -41,7 +41,12 @@ jobs:
       - run: go get golang.org/x/tools/cmd/cover
       - checkout
       - run: go mod tidy
-      - run: go vet ./...
+      - run:
+          name: Check Vet
```

Comment:
Reading again, I believe that this should be in `lint` area.

Reply:
Yes, I am confused about that part I'll shift this to the lint section.